### ISSUE TITLE
better error message for `cannot import` packages in core

### ISSUE THREAD

**Issue by bobzhang** [moonbitlang/moon#1158]
```
error: /private/var/folders/jb/64x3jtc51qbfk3y1m9f3yvyc0000gn/T/maria-test-CZkxDLF/json_parser/moon.pkg.json: cannot import `moonbitlang/core/buffer` in `username/json_parser`, no such package
/private/var/folders/jb/64x3jtc51qbfk3y1m9f3yvyc0000gn/T/maria-test-CZkxDLF/json_parser/moon.pkg.json: cannot import `moonbitlang/core/strconv` in `username/json_parser`, no such package
```

relevant code:
```
                if !self.packages.contains_key(&imported) {
                    errors.push(format!(
                        "{}: cannot import `{}` in `{}`, no such package",
                        pkg.root_path.join(MOON_PKG_JSON).display(),
                        imported,
                        pkg.full_name(),
                    ));
                }
```

`moonbitlang/core` comes from builtin, maybe we should give a warning and skip it?
we need check if it is aliased

**Comment by lynzrand** [comment]
I personally would like to still fail the build for this (this is a wrong import, #1035 is not implemented), but instead of a generic error, print that `core` is already imported.
